recipes/peka/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "PEKA" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/ulelab/peka/archive/refs/tags/v0.1.2.tar.gz"
```

Comment:
```suggestion
  url: "https://github.com/ulelab/peka/archive/refs/tags/v{{ version }}.tar.gz"
```

Reply:
Oh, many thanks for catching this, it should be v0.1.2.